apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -323,6 +332,39 @@ class TrygdetidServiceImpl(
             logger.info("Opprettet ${it.size} trygdetider for behandling=${behandling.id}")
         }
 
+    private fun populertrygdetidFraPesys(
+        trygdetid: Trygdetid,
+        pesystt: TrygdetidsgrunnlagUfoeretrygdOgAlderspensjon,
+    ): Trygdetid {
+        val mappedAP =
+            pesystt.trygdetidAlderspensjon?.trygdetidsgrunnlag?.trygdetidsgrunnlagListe?.map {
+                mapPesysTrygdetidsgrunnlag(it)
+            } ?: emptyList()
+
+        val mappedUfoere =
+            pesystt.trygdetidUfoeretrygdpensjon?.trygdetidsgrunnlag?.trygdetidsgrunnlagListe?.map {
+                mapPesysTrygdetidsgrunnlag(it)
+            } ?: emptyList()
+
+        return trygdetid.copy(trygdetidGrunnlag = mappedAP + mappedUfoere)
```

Comment:
Er vi sikre på at disse periodene ikke overlapper hverandre? Spør av egen uvitenhet her, mulig svaret er at de selvsagt ikke gjør det

Reply:
Vet ikke, det kommer vel ann på hva sb har gjort i PESYS. Men er det krise for oss om de er det da?

Reply:
Vil det egentlig validere? Tror ikke vi kan ha overlappende perioder. Støtter reglene våre det?

Reply:
Vi er sikre på at periodene ikke overlapper!